### ISSUE TITLE
⌨ (in progress) Skills icons without alt - My Profile page

### ISSUE THREAD

**Issue by MaddyDaigle** [GCTC-NTGC/gc-digital-talent#4996]
IN PROGRESS!!!

## ❌ Current behaviour (issue)
The icons in the list of experience and skills don’t currently have alt text.  Since this is providing information to only visual users (the type of skill) - this should have alt text associated with it.

**WCAG 2.1 Criteria:**

### How to reproduce
1. Environment (prod or UA?) Prod, https://talent.canada.ca/en/users/.../profile
2. Which browser and/or Assistive Technology? Chrome
3. Step-by-step instructions explaining how to reproduce the problem:
Go to the profile page, enter in an experience and skill under "My Skills and Experience".  Return to the profile page.  You'll notice that the list of experiences has icons next to it. 
<img width="898" alt="Screen Shot 2022-11-29 at 1 12 27 PM" src="https://user-images.githubusercontent.com/1794660/205371999-066ce8c5-96c1-4b29-927a-5be1f5e99e65.png">

## ✔ Expected behaviour (fix)
The screen reader should read out alt text for the icon that identifies the type of skill.
- Briefcase: Work experience
- Book: Education experience
- People: Community experience
- Light bulb: Personal experience
- Star: Award experience

### Acceptance Criteria
A set of assumptions which, when tested, verify that the accessibility requirement was met.
- [ ]
- [ ]

**Comment by RM-1978** [comment]
A UAT review reveals that this change has been implemented there - Closing the issue
